ckanext/harvest/logic/action/get.py

```diff
@@ -450,14 +453,19 @@ def harvest_get_notifications_recipients(context, data_dict):
             'capacity': 'admin'
         })
 
+        # Get access to email address by running action as admin user
+        context['user'] = 'admin'
         for member in members:
-            member_details = p.toolkit.get_action(
-                'user_show')(context, {'id': member[0]})
+            member_details = p.toolkit.get_action('user_show')(context, {
+                   'id': member[0],
+                   'include_plugin_extras': True
```

Comment:
I am using the 2.9 branch on ckan, and I do not get back the email address without setting `include_plugin_extras ` to True.    But it could be an unusual user configuration causing this, I am not sure.

Reply:
I will go back and check again using the suggested change for setting the user context.   This could make a difference.

Reply:
After using the suggested user context,  here is what I get for `member_details` when `include_plugin_extras == False`:
```
{'id': 'a22a75a1-0acb-413a-ab8d-d9cf875e5c7f', 
'name': 'admin', 
'fullname': None, 
'created': '2022-05-13T00:23:35.996281', 
'about': None, 
'activity_streams_email_notifications': False, 
'sysadmin': True, 
'state': 'active', 
'image_url': None, 
'display_name': 'admin', 
'email_hash': '296caa4405738709b57e43532e4232dd', 
'number_created_packages': 206, 
'apikey': None, 
'email': 'devnull00@ucar.edu', 
'image_display_url': None}
```
 It is only when I set `include_plugin_extras == True` that I get the additional email field.   

The problem, I believe, is that we have a user seeking an email address that is not their own.   The logic is different than when a user seeks for their own email address in `user_show`.

Reply:
It looks like the email address is there now without the extra flag.   I will remove it.